src/App.jsx

```diff
@@ -14,70 +16,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [isSelected, setIsSelected] = useState(true);
```

Comment:
you don't need this state
just check `selectedGood` below - an empty string is a falsy value: https://www.freecodecamp.org/news/falsy-values-in-javascript/